storage/s3/src/main/java/io/aiven/kafka/tieredstorage/storage/s3/S3StorageConfig.java

```diff
@@ -74,7 +74,7 @@ public class S3StorageConfig extends AbstractConfig {
     public static final String AWS_SECRET_ACCESS_KEY_CONFIG = "aws.secret.access.key";
     private static final String AWS_SECRET_ACCESS_KEY_DOC = "AWS secret access key. "
         + "To be used when static credentials are provided.";
-    public static final String DISABLE_AWS_CERT_CHECKING_CONFIG = "storage.disableAWSCertChecking";
+    public static final String DISABLE_AWS_CERT_CHECKING_CONFIG = "disableAWSCertChecking";
```

Comment:
nit: to follow dotted notation:
```suggestion
    public static final String DISABLE_AWS_CERT_CHECKING_CONFIG = "aws.disable.cert.checking";
```